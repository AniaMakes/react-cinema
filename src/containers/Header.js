import React from 'react';

const Header = () => (
		<header className='main-header'>
			<h1>
				Find any movie!
			</h1>
		</header>
	)

export default Header;