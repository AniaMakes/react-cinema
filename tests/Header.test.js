import React from 'react';
import Header from '../src/containers/Header';
import renderer from 'react-test-renderer';

describe('Header', () => {
	it('correctly renders the Header', () => {
		const tree = renderer.create(<Header />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
