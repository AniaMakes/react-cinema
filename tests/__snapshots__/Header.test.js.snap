// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches the snapshot 1`] = `
<header
  className="main-header"
>
  <h1>
    Find any movie!
  </h1>
</header>
`;
